src/components/SelectionList/TransactionListItem.tsx

```diff
@@ -61,6 +62,7 @@ function TransactionListItem<TItem extends ListItem>({
     const isFromExpenseReport = transactionItem.reportType === CONST.REPORT.TYPE.EXPENSE;
     const date = TransactionUtils.getCreated(transactionItem as OnyxEntry<Transaction>, CONST.DATE.MONTH_DAY_ABBR_FORMAT);
     const amount = TransactionUtils.getAmount(transactionItem as OnyxEntry<Transaction>, isFromExpenseReport);
+    const taxAmount = TransactionUtils.getAmount(transactionItem as OnyxEntry<Transaction>);
```

Comment:
This is not correct as it'll get the full transaction amount, we should use `getTaxAmount`
```suggestion
    const taxAmount = TransactionUtils.getTaxAmount(transactionItem as OnyxEntry<Transaction>);
```